tasks/30_people-in-bus/people-in-bus.js

```diff
@@ -0,0 +1,13 @@
+//  I'm repeating myself, probably it could be refactored
+//  but I don'know how :)
+
+export const peopleInBus = ( arr ) => {
+  const peopleIn = arr.map( ( item ) => item[0] )
```

Comment:
it will be better to use one reduce in which you will add people who in and minus who out :)

Reply:
I have no idea how I could use one reduce method, how I can I get the values of elements of multidimensional array without using map first. Or maybe I don't understand your comment. I could get rid of variables but this is not what you wanted :) `
return arr.map( ( item ) => item[0] ).reduce( ( prev, next ) => prev + next, 0 ) -  arr.map( ( item ) => item[1] ).reduce( ( prev, next ) => prev + next, 0 );` It looks like shit :)